NiBAx/__init__.py

```diff
@@ -9,26 +9,39 @@
 import argparse
 import os, sys
 from NiBAx.mainwindow import MainWindow
+from NiBAx.NiBAxCmdApp import NiBAxCmdApp
 
 def main():
     parser = argparse.ArgumentParser(description='NiBAx Data Visualization and Preparation')
     parser.add_argument('--data_file', type=str, help='Data file containing data frame.', default=None, required=False)
     parser.add_argument('--harmonization_model_file', type=str, help='Harmonization model file.', default=None, required=False)
     parser.add_argument('--SPARE_model_file', type=str, help='Model file for SPARE-scores.', default=None, required=False)
+    parser.add_argument('--harmonize', type=str, help='Do harmonization or not.', default=None, required=False)
+    parser.add_argument('--compute_spares', type=str, help='Compute SPARE-scores or not.', default=None, required=False)
+    parser.add_argument('--output_file_name', type=str, help='Name of the output file with extension.', default=None, required=False)
+    parser.add_argument('--nogui', type=str, help='Run the application with or without GUI', default=None, required=False)
```

Comment:
```suggestion
parser.add_argument("-nogui", action="store_true", help="…")
# args.nogui becomes True if flag is set
```

Reply:
@AbdulkadirA Thanks for the suggestion.
Although it is self-explanatory, it is still good to add a short help message. What do you think?

Reply:
@ashishsingh18  A message like "The 'nogui' option will not launch the graphical user interface. This is useful for data processing without wisualization."